README.rst

```diff
@@ -49,12 +49,19 @@ The -L flag can be used to whitelist certain words that are comma-separated plac
 
     codespell -S, --skip=
 
-Comma-separated list of files to skip. It accepts globs as well.  Examples:
+The -S (--skip) flag is used to skip over a comma-separated list of files and/or directories. It accepts globs as well. **Important note:** Do not append a '/' to a directory you want to skip. This currently renders the entry ineffective.
 
-* to skip .eps & .txt files, invoke ``codespell --skip="*.eps,*.txt"``
+Examples:
 
-* to skip directories, invoke ``codespell --skip="./src/3rd-Party,./src/Test"`` ::
+* Skip .eps & .txt files, invoke one of::
 
+    codespell --skip="*.eps,*.txt"
+    codespell -S *.eps,*.txt
```

Comment:
I'm not sure listing the short and long version as an example adds much. Power users will understand the relationship between them without needing an example, and normal users just need something that works. Having a few more examples would be more helpful if anything (e.g. .txt files and all files in a particular folder).

Reply:
I didn't understand this at first but learned after usage and contributing to the project. I look at this as a way to turn amateurs in to power users. I vote to keep this.

Reply:
Personally I'd worry it just confuses them, especially given you've also changed the short mode syntax to drop the equals and quote. If you did the same thing with the long one, it's only four more characters to type "-kip" and it's more obvious what you're doing on the command line. Whereas I'd worry it's a lot more noise if each example is listed twice. If you really want to do it, personally I'd think a line at the end explaining how to use short arguments would be more effective (or probably link to somewhere else that does so already, as e.g. short flag chaining "grep -iIR" is where the benefit can really start to kick in. You haven't actually explained the other usage, you've just written something that looks similar, it wants a comment saying, this can also be done using the short flags as...

Reply:
I still personally disagree with cluttering the examples like this.